File: CH3-desicion-sructure-and-boolean-logic/EX8-shipping-charges/program-code/shippingCharges.py
weightPackage = input("Enter a weight of a package :")

if weightPackage <= 2:
    shippingCharges = 1.10
else:
    if weightPackage > 2 and weightPackage <= 6:
        shippingCharges = 2.20
    else:
        if weightPackage > 6 and weightPackage <= 10:
            shippingCharges = 3.70
        else:
            shippingCharges = 3.80


print('Shipping charges : {}'.format(shippingCharges))

File: CH3-desicion-sructure-and-boolean-logic/EX8-shipping-charges/program-code/shippingCharges.js
const input = require("prompt-sync")();

let weightPackage;
let shippingCharges;

weightPackage = input("Enter a weight of package :");

if (weightPackage <= 2) {
  shippingCharges = 1.1;
} else {
  if (weightPackage > 2 && weightPackage <= 6) {
    shippingCharges = 2.2;
  } else {
    if (weightPackage > 6 && weightPackage <= 10) {
      shippingCharges = 3.7;
    } else {
      shippingCharges = 3.8;
    }
  }
}

console.log("Shipping charges : $" + shippingCharges);
